lms/static/sass/course/_info.scss

```diff
@@ -56,6 +61,85 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    .upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      border-style: solid;
+      border-color: $uxpl-gray-background $uxpl-gray-background $uxpl-gray-background $uxpl-green-base;
+      border-width: 1px 1px 1px 10px;
```

Comment:
This is going to have RTL issues because the left border is different from the right. I think you need to use "@include border-left" and "@include border-right", but I would run that by a FED person.